app/templates/package.json

```diff
@@ -2,19 +2,25 @@
   "name": "<%= moduleName %>",
```

Comment:
Sorry, that damn `,`. I kept making the same mistake over and over every time I removed a field or changed the JSON a little bit.